x/incentive/types/msg.go

```diff
@@ -15,6 +15,7 @@ var (
 	_ sdk.Msg = &MsgClaimDelegatorReward{}
 	_ sdk.Msg = &MsgClaimSwapReward{}
 	_ sdk.Msg = &MsgClaimSavingsReward{}
+	_ sdk.Msg = &MsgClaimEarnReward{}
```

Comment:
probably as a separate PR, but wanted to note that we should also add legacy msg type check here as well.
```
_ legacytx.LegacyMsg = &MsgClaimEarnReward{}
```

Reply:
just added to this pr since they all seem to already comply to the interface so it's a small change